primitives/core/src/assertion.rs

```diff
@@ -177,6 +177,9 @@ pub enum Assertion {
 	// ----- end polkadot decoded 2023 -----
 
 	Achainable(AchainableParams),
+
+	// For EVM Version Early Bird
+	A20(ParameterString),									// (participant_account)
```

Comment:
Does it mean i can provide any account ? For example the one i doesn't own ?

Reply:
@kziemianek Nice question. We have collaboration with colleague who prepared `https://archive-test.litentry.io`. We don't process the incoming parameter. Whatever comes in will be forwarded to get-request. Because all the steps are in house. Wdyt?

Reply:
I think it doesn't answer the question 🙃 .

What prevents me from getting this VC with value `true` ?
If my friend joined evm campaing and shares his address with me, does it mean i can get this VC aswell ?

Reply:
Thanks. You are right. We should not allow any input parameter here. Loop inside the IDGraph.

Reply:
@BillyWooo  wouldn't prime identity be sufficient ?

Reply:
@kziemianek  yes. You are right. We just had a discussion. Prime account is enough.

Reply:
Yea I don't think we need any parameter here